src/cloudabi-run/yaml_file_descriptor_factory.h

```diff
@@ -18,8 +18,8 @@ namespace cloudabi_run {
 class YAMLFileDescriptorFactory
     : public yaml2argdata::YAMLFactory<const argdata_t *> {
  public:
-  YAMLFileDescriptorFactory(YAMLFactory<const argdata_t *> *fallback)
-      : fallback_(fallback) {
+  YAMLFileDescriptorFactory(YAMLFactory<const argdata_t *> *fallback, const int execfd)
```

Comment:
Could you remove the `const` keyword here? It typically doesn't make any sense to make function arguments themselves const. References and pointers to const objects make sense, but having the argument itself be `const` doesn't tend to be useful.